docs/markdown/Using Pants/validating-dependencies.md

```diff
@@ -262,7 +266,9 @@ The previous example, using this alternative syntax for the selectors, would loo
 Glob syntax
 -----------
 
-The visibility rules support a basic glob syntax, using `*` as a match anything non-recursively with the `**` variant being for a recursive match anything. All rules are matched until the end of the path it is being applied to, so if there is not trailing wildcard (`*` or `**`) the end of the rule glob will match the end of the path. This allows for matching on file types/names regardless of where in the project tree they are:
+The visibility rules is all about matching globs. There are two primary kinds of globs supported, the simple glob syntax and the path glob syntax. Simple globs only supports `*` as a match anything and is otherwise case sensitive and are used for all glob values except for the `path` part of a __target rule spec__. The path glob syntax is described further below.
+
+Path globs have two wildcards, the `*` as a match anything non-recursively and the `**` for a recursive match anything. All globs are matched until the end of the value it is being applied to, so if there is no trailing wildcard (`*` or `**`) the end of the path glob it will match until the end of the path. This allows for matching on file names regardless of where in the project tree they are:
```

Comment:
```suggestion
Path globs have two wildcards, the `*` to match anything non-recursively and the `**` to match anything recursively. Each glob is matched until the end of the value it is being applied to, so if there is no trailing wildcard (`*` or `**`) on the end of the path glob, it will match to the end of the path. This allows for matching on file names regardless of where in the project tree they are:
```